### PR TITLE
Initial Implement of FST Online Monitor

### PR DESCRIPTION
**Pull request by sunxuhit**
The FST Online Monitor code (fstBuilder) is based on istBuilder with some updates due to different geometry.
The code is tested on my local RCF account with daq file took by Run Control.

### CODE REVIEW

**Comment by sunxuhit** [comment]
Dear Expert,

Not sure what is this CI Build/ROOT5 failure means here. 
I could compiled the fstBuilder at my rcf account with ROOT 5.34 without any error.
Could you please point me where is the problem?

Xu

File: OnlTools/Jevp/StJevpBuilders/fstBuilder.h [hunk 1]
```diff
@@ -0,0 +1,651 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "JevpBuilder.h"
+#include "DAQ_READER/daqReader.h"
```
Comment (plexoos) [inline OnlTools/Jevp/StJevpBuilders/fstBuilder.h]: Try to forward declare instead

```suggestion
class daqReader;
```

**Comment by plexoos** [comment]
Something is wrong with the header file as rootcint chokes on it for some reason. I would suspect the unions but you say it compiles in a different environment...

**Review by jml985 — APPROVED** [review]
I've tested the changes and they produce appropriate plots, the resource requirements are not out of bounds, and the architecture seems correct.

**Pull request merged**